dev-python/logfury/logfury-0.1.2-r1.ebuild

```diff
@@ -18,4 +19,7 @@ KEYWORDS="~amd64 ~x86"
 RDEPEND="dev-python/six[${PYTHON_USEDEP}]"
 
 DEPEND="${RDEPEND}
-	dev-python/setuptools[${PYTHON_USEDEP}]"
+	dev-python/setuptools[${PYTHON_USEDEP}]
```

Comment:
This is redundant.

Reply:
Removed redundant setuptools requirement.